ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchasesHybridCommonTests.swift

```diff
@@ -40,137 +49,136 @@ class PurchasesHybridCommonTests: QuickSpec {
         }
 
         context("logIn") {
-//            it("passes the call correctly to Purchases") {
```

Comment:
re-enabled these tests, with the updated code from https://github.com/RevenueCat/purchases-hybrid-common/pull/113